app/graphql/mutations/comment_survey.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: false
+
+module Mutations
+  class CommentSurvey < BaseMutation
+    argument :survey_id, ID, required: true
+    argument :message, String, required: true
+
+    type Types::StatusType
+
+    def resolve(survey_id:, message:)
+      survey = Survey::Poll.find_by(id: survey_id)
+
+      return error_status("Access not permitted: Survey missing") if survey.blank?
+      return error_status("Access not permitted: Survey already archived") if survey.archived?
+
+      comment = survey.comments.build(message: message)
+      return error_status("Access not permitted: Comment invalid, #{comment.error_messages}") unless comment.valid?
+
+      begin
+        comment.save
+        OpenStruct.new(id: survey_id, status: "comment successfully", status_code: 200)
+      rescue => e
+        error_status("Error on commenting: #{e}")
```

Comment:
das hier braucht noch einen status_code, vermutlich 500, da es kein 403 not permitted ist an der stelle

Reply:
ebenso bei `VoteForSurvey`, kümmere mich drum

Reply:
done in ffe0ef9df9813a4db8afc55ee3caab5b9284fc0d